ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageBackgroundView.swift

```diff
@@ -32,6 +48,8 @@ class NewTabPageBackgroundView: UIView {
       return .init(rgb: 0x3b3e4f)
     }
 
+    layer.addSublayer(playerLayer)
+
```

Comment:
Lets move this to `setupPlayerLayer` so we dont bother adding it unless its actually going to be used